src/Microsoft.TemplateEngine.Cli/ChoiceTemplateParameter.cs

```diff
@@ -80,7 +80,9 @@ protected override Option GetBaseOption(IReadOnlyList<string> aliases)
                 AllowMultipleArgumentsPerToken = AllowMultipleValues
             };
 
-            option.FromAmongCaseInsensitive(Choices.Keys.ToArray());
+            // empty string for the explicit unset option
+            option.FromAmongCaseInsensitive(Choices.Keys.ToArray(), string.Empty);
```

Comment:
nit: was a bit confused what it `string.Empty` here, named parameter will be helpful